src/components/Dissolution/CompleteAffidavit.vue

```diff
@@ -374,7 +374,7 @@ export default class CompleteAffidavit extends Mixins(CommonMixin, DocumentMixin
           name: this.uploadAffidavitDoc.name,
           lastModified: this.uploadAffidavitDoc.lastModified,
           size: this.uploadAffidavitDoc.size
-        }
+        } as File
```

Comment:
Out of curiosity, was this causing issues?

Reply:
Yup.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/83095d2a-5709-4e1e-a98d-8b239b69017e)